rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
```

Comment:
[ Optional Changes ]

For a getter and setter, you could use the attr_accessor as it does bother, getting and setting a value.
`attr_accessor :date, :book, :person`